optin/reporting/test/routers/api.spec.js

```diff
@@ -170,4 +170,21 @@ describe('API Route', () => {
 			});
 		});
 	});
+
+	describe('wildcard route', () => {
+		// Arrange:
+		const randomPath = ['/test', '/api/completed/random', '/api/requests/random'];
+
+		it('returns html index page', async () => {
+			/* eslint-disable no-await-in-loop */
+			for (let i = 0; i <= randomPath.length - 1; ++i) {
+				// Act:
+				const response = await request(app).get(randomPath[i]);
+
+				// Assert:
+				expect(response.status).to.equal(200);
```

Comment:
why is this 200 instead of 404 (not found) ?

Reply:
yeah because I had added `*` in the route, so technically whatever route, it should hit the HTML page.

what do you think about this?

https://github.com/symbol/product/blob/1dc3f89bb5ae03300b7997cb824c4506b276cd89/optin/reporting/server.js#L20

Reply:
can you return 404 if file does not exist?

Reply:
sure thing, let me add this test.